src/components/MultiHopTrade/hooks/useGetTradeQuotes/useGetTradeQuotes.tsx

```diff
@@ -143,7 +143,11 @@ export const useGetTradeQuotes = () => {
   const sellAssetUsdRate = useAppSelector(s => selectUsdRateByAssetId(s, sellAsset.assetId))
 
   const isSnapshotApiQueriesPending = useAppSelector(selectIsSnapshotApiQueriesPending)
-  const votingPower = useAppSelector(selectVotingPower)
+  const votingPowerParams: { feeModel: ParameterModel } = useMemo(
```

Comment:
can you explain the need for this? would a static value in `useAppSelector` really cause unnecessary re-renders?

Reply:
Absolutely - `const votingPowerParams = {}` would evaluate to a new reference every render because it's an (object) array, hence will cause more re-renders/recomputations

Reply:
could be done as a const outside of the component as well then I would assume right? (just for extra clarificaiton)

Reply:
Actually since this one is static, might as well yes! c32f1dc1a7cfe60a8aa55c7715f7233308d33e91

This would avoid an extra memo call for all the static ones 🧠